packages/start/package.json

```diff
@@ -134,7 +134,6 @@
   },
   "dependencies": {
     "@tanstack/react-cross-context": "workspace:^",
```

Comment:
It may be worth checking out, whether we are actually using the `react-cross-context` package and possibly remove it.